python/GafferSceneUI/LightEditor.py

```diff
@@ -302,48 +309,69 @@ def __buttonDoubleClick( self, pathListing, event ) :
 
 	def __keyPress( self, pathListing, event ) :
 
-		if event.key == "Return" or event.key == "Enter" :
-			self.__editSelectedCells( pathListing )
+		if event.modifiers == event.Modifiers.None_ :
 
-			return True
+			if event.key == "Return" or event.key == "Enter" :
+				self.__editSelectedCells( pathListing )
 
-		return False
+				return True
 
-	def __editSelectedCells( self, pathListing ) :
+			if event.key == "D" :
+				self.__disableEdits( pathListing )
 
-		selection = pathListing.getSelection()
+				return True
 
-		columns = pathListing.getColumns()
+			if event.key == "Delete" :
+				self.__removeAttributes( pathListing )
```

Comment:
This needs the same "is the whole selection deletable?" check that we have in the menu building code. But I've made a suggestion in `__removableAttributeInspections` that would make this moot. The same applies to the disable shortcut...

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4948/commits/182b1e8f711fbdaf4e3096a88b86bd555810344f and https://github.com/GafferHQ/gaffer/pull/4948/commits/00da3c790ea52e7635bb5cfa13fdb92cf611012e